ludwig/utils/data_utils.py

```diff
@@ -327,13 +331,15 @@ def save_hdf5(data_fp, data):
             h5_file.create_dataset(column, data=numpy_dataset[column])
 
 
-def load_hdf5(data_fp):
+def load_hdf5(data_fp, clean_cols: bool = False):
```

Comment:
Just a reminder to make sure we want to flip the default value for `clean_cols` here compared to the caller.